### DISCUSSION TITLE
Multiple micros per project

### DISCUSSION THREAD

**Discussion by takecare** [deta/cloud-docs#294] [Help] [answered]
I've noticed that by default any new micros fall under the `default` project. I've gone through the docs but can't find anything related to having more than one micro per project. Is this not possible?

**Comment by jnsougata** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/53375272/176039685-f4a8744c-252a-4cf5-b837-3c454d8f7db3.png)
and yes you can have more than one micro per project
